tutorials/attribution/before-you-start.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Before you start
+position: 2
+---
+
+Marketing Attribution Analysis can be done in many ways depending on your business and needs. Our aim has always been to make it flexible for users to make changes from selecting a different data source to reshaping how the customer journey to conversion will be adjusted for the analysis. We suggest taking the time initially, before running the model and also readjust the configurations later on to fit your needs. 
```

Comment:
"We suggest taking the time initially, before running the model and also readjust the configurations later on to fit your needs."

Could be clearer - I assume it means to take the time to read the documentation? 

"We suggest taking the time to read the documentation before running the model and also readjust the configurations later on to fit your needs."